src/index.html

```diff
@@ -13,10 +13,14 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
```

Comment:
The stylesheet link is pointing to a `.scss` file. Browsers do not understand SCSS directly; it needs to be compiled to CSS. Ensure that `styles/index.scss` is compiled to `styles/index.css` and update the link to point to the `.css` file.